docs/api/navigators/StackNavigator.md

```diff
@@ -111,6 +111,7 @@ Title string used by the back button on iOS, or `null` to disable label. Default
 #### `headerBackImage`
 
 The image source (either a remote URL `{{uri: 'path/to/image'}}` or a local file resource  `{require('path/to/image')}`) used by the back button.
+Defaults to './views/assets/back-icon.png'
```

Comment:
Done. I think link is not good idea because of different images for ios/android..